### ISSUE TITLE
[BUG]  The error associated with the inability to use the installed tools with the setuptools 60.9.0 version (Released: 14.02.2022 00:00 ) Command not found

### ISSUE THREAD

**Issue by Sinyuk** [pypa/setuptools#3110]
### setuptools version

setuptools == 60.9.0

### Python version

Python 3.7-3.9

### OS

Linux, Windows

### Description

The error associated with the inability to use the installed tools with the setuptools 60.9.0 version (Released: 14.02.2022 00:00 ) Command was not found. Work with any version to 60.6.0.

After installation, the package is installed, but not when called, it says that the command was not found

### Expected behavior

I expect that the tools will work

### How to Reproduce

1. Install setuptools 60.9.0 version or use pip install --force-reinstall <path>
2. Call the following tool
3. Get error "Command not found"

### Output

```console
$ test
bash: test: command not found

```

**Comment by abravalheri** [comment]
Hi @Sinyuk, thank you for bringing this issue up, however I don't know if I understand your problem description and the steps you provided to reproduce the problem...

I am afraid `setuptools` never provided a CLI command called `test` that you could run in your own. I can validate this belief by running:

```bash
virtualenv .venv
.venv/bin/pip install 'setuptools==60.6.0'
ls .venv/bin | grep test
# ==> there is no `test` executable in `.venv/bin
```

Could you please expand more your problem description and clarify that?
Also could you provide a standalone (and minimal) method to replicate the problem? For example you can provide a couple of bash commands that show the fault in a virtual environment, or even a Dockerfile...

**Comment by Sinyuk** [comment]
Dear, @abravalheri 
**How to Reproduce:**
1. git clone https://github.com/epam/aws-syndicate
2. Add the 'setuptools==60.9.0' field to install_requires in setup()        
3. pip install --force-reinstall -e .
4. execute 'syndicate'


Maybe resolved in 60.9.1 - _Fixed issue where string-based entry points would be omitted._ 
The 60.9.1 version works as expected.

**Comment by abravalheri** [comment]
Ah perfect!

Thank you very much @Sinyuk for the clarification @Sinyuk.
Looking at the project you shared I can see that it relies on [`entry_points` being specified as a string](https://github.com/epam/aws-syndicate/blob/3f95c645602c1196fa155a939f3d16ffc0d93021/setup.py#L43). Indeed v60.9.0 had problems with those, as reported in #3103.

This bug was fixed in v60.9.1. So I will go ahead and close this issue.
Please let us know if the problem comes back in the future.

**Comment by Sinyuk** [comment]
Ok, thanks
